doc/man3/SSL_CONF_cmd.pod

```diff
@@ -535,6 +535,14 @@ B<SSL_OP_ENABLE_KTLS>.
 B<StrictCertCheck>: Enable strict certificate checking. Equivalent to
 setting B<SSL_CERT_FLAG_TLS_STRICT> with SSL_CTX_set_cert_flags().
 
+B<KTLSTxZerocopySendfile>: use the zerocopy TX mode of sendfile(), which gives
+a performance boost when used with kTLS hardware offload. Note that invalid TLS
+records might be transmitted if the file is changed while being sent. This
```

Comment:
> Note that invalid TLS records might be transmitted if the file is changed while being sent.

This should be emphasized A LOT more. Big flags around it with blinking LEDS.

Reply:
Any specific suggestions how to rephrase it?

Considering that `sendfile()` manpage explicitly prohibits modifying the file:

```
       If out_fd refers to a socket or pipe with zero-copy support,
       callers must ensure the transferred portions of the file referred
       to by in_fd remain unmodified until the reader on the other end
       of out_fd has consumed the transferred data.
```

and OpenSSL on FreeBSD has the same limitation, but it's even worse, because it is the default, can't be turned off and not documented anywhere in OpenSSL, I think the Linux side is documented good enough.

Reply:
The problem is that users aren't necessarily looking at Linux documentation while reading the OpenSSL documentation.  It should probably made its own paragraph, possibly in a WARNINGS section, or with the text "WARNING" preceding it. Please look in `doc/man3` for further examples.
```
If this option is set, SSL_sendfile() will use the zerocopy TX mode, which gives
a performance boost when used with kTLS hardware offload. 
This option automatically enables the B<-sendfile> option.
Note that kTLS sendfile on FreeBSD always runs in the zerocopy mode.

WARNING: Invalid TLS records might be transmitted if the file is changed while being sent.
Please refer to the L<sendfile(3)> documentation.
```
Or:
```
=head1 WARNINGS

Invalid TLS records might be transmitted if the file is changed while being sent.
Please refer to the L<sendfile(3)> documentation.

Reply:
This warning is not Linux-specific. FreeBSD suffers from the same limitation, moreover it can't even be disabled. If Linux users don't read Linux documentation, then FreeBSD users don't do it either, so OpenSSL should have an even more prominent warning for FreeBSD, shouldn't it?

That is, it doesn't make sense to me to add the Linux warning when there is no FreeBSD warning, and documenting FreeBSD limitations is outside the scope of my pull request. Besides, it might be possible to document it somewhere in a single place for both Linux and FreeBSD.

Reply:
Understood. Note that my suggestions were not Linux nor FreeBSD specific.

Reply:
This still hasn't been addressed. I will also note that the phrasing below "This option only makes on Linux", is probably better as "This option only applies to Linux."